pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/LoadReportNetworkLimit.java

```diff
@@ -49,8 +55,8 @@ public void checkLoadReportNicSpeed() throws Exception {
         LoadManagerReport report = admin.brokerStats().getLoadReport();
 
         if (SystemUtils.IS_OS_LINUX) {
-            assertEquals(report.getBandwidthIn().limit, 5.4 * 1024 * 1024);
-            assertEquals(report.getBandwidthOut().limit, 5.4 * 1024 * 1024);
+            assertEquals(report.getBandwidthIn().limit, nicCount * 5.4 * 1024 * 1024);
```

Comment:
what is the nicCount when it's not Linux ?

Reply:
`0`, it's only set if `SystemUtils.IS_OS_LINUX`, that's in the setup method.

Reply:
I know that this isn't a beautiful solution. Rewriting the test in a different way would be useful with time. I'm wondering why the current CI doesn't fail with this test. Perhaps it gets skipped somehow. 

The failure shows up with the Pulsar CI GitHub Actions workflows refactoring work related to [PIP Changes to GitHub Actions based Pulsar CI](https://docs.google.com/document/d/1FNEWD3COdnNGMiryO9qBUW_83qtzAhqjDI5wwmPD-YE/edit?usp=sharing) where this is blocking the progress.

[Example failure](https://github.com/lhotari/pulsar/runs/1888404224?check_suite_focus=true#step:10:1151) (use CTRL/CMD click for opening link to get the focus to scroll to the error)

I'm currently iterating on the workflow refactoring and I'm soon done with unit tests. I'll be creating PRs from the experiment I'm working on, it's simple a PoC currently. 

This is the run with the fix of this PR included: https://github.com/lhotari/pulsar/actions/runs/561829678 (currently in progress) . It's going to be a lot faster than the current workflow.